LeaderboardBackend/Services/ILeaderboardService.cs

```diff
@@ -0,0 +1,12 @@
+using LeaderboardBackend.Models;
+
+namespace LeaderboardBackend.Services
+{
+    public interface ILeaderboardService
+    {
+        Task<Leaderboard?> GetLeaderboard(long id);
+
+        Task<List<Leaderboard>> GetLeaderboards(long[] ?ids = null);
```

Comment:
nit: slight inconsistency in the line break choices

Reply:
Don't think I can see this in my editor.. would you mind fixing this?

Reply:
We can leave it and figure out what happened when this is merged.